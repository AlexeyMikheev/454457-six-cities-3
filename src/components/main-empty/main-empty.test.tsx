import * as React from "react";
import {mount} from "enzyme";
import MainEmpty from "./main-empty";

it(`MainEmpty snapshot`, () => {

  const div = document.createElement(`div`);
  document.body.appendChild(div);

  const mainEmptyComponent = mount(<MainEmpty />, {attachTo: div});

  expect(mainEmptyComponent.getDOMNode()).toMatchSnapshot();
});
