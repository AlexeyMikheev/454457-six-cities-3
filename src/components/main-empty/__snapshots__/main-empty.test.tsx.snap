// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainEmpty snapshot 1`] = `
Array [
  <div
    class="cities__places-container cities__places-container--empty container"
  >
    <section
      class="cities__no-places"
    >
      <div
        class="cities__status-wrapper tabs__content"
      >
        <b
          class="cities__status"
        >
          No places to stay available
        </b>
        <p
          class="cities__status-description"
        >
          We could not find any property availbale at the moment in Dusseldorf
        </p>
      </div>
    </section>
    <div
      class="cities__right-section"
    />
  </div>,
]
`;
